graphql_compiler/macros/__init__.py

```diff
@@ -176,7 +181,16 @@ def get_schema_for_macro_definition(schema):
     Returns:
         GraphQLSchema usable for writing macros
     """
-    raise NotImplementedError()  # TODO(bojanserafimov): Implement
+
+    required_macro_directives = (MacroEdgeDefinitionDirective, MacroEdgeTargetDirective)
```

Comment:
@bojanserafimov Why don't we just add macro directives in all schemas?

Reply:
We don't want the graphql editor to tell the user to maybe define a macro in the middle of their query, because that doesn't work.